src/hotspot/share/runtime/arguments.cpp

```diff
@@ -521,6 +521,8 @@ static SpecialFlag const special_jvm_flags[] = {
   { "InitialRAMFraction",           JDK_Version::jdk(10),  JDK_Version::undefined(), JDK_Version::undefined() },
   { "AllowRedefinitionToAddDeleteMethods", JDK_Version::jdk(13), JDK_Version::undefined(), JDK_Version::undefined() },
   { "FlightRecorder",               JDK_Version::jdk(13), JDK_Version::undefined(), JDK_Version::undefined() },
+  { "SuspendRetryCount",            JDK_Version::jdk(16), JDK_Version::jdk(17), JDK_Version::jdk(18) },
+  { "SuspendRetryDelay",            JDK_Version::jdk(16), JDK_Version::jdk(17), JDK_Version::jdk(18) },
```

Comment:
I think both 'jdk(16)' values here need to be 'jdk(17)' since
we didn't deprecate these option in JDK16. Or they might
need to be 'JDK_Version::undefined()' since we didn't
deprecate these options before obsoleting them.
@dholmes-ora will know for sure.

Reply:
Fixed with undefined() as David said.